harvard-cape-peninsula-university-of-technology.csl

```diff
@@ -109,8 +109,8 @@
   <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
     <layout prefix="(" suffix=")" delimiter="; ">
       <group>
```

Comment:
I tested out the deliminator suggestion on csl.mendeley.com, but it doesn't give me the result that I want: it removed the comma completely from the citation. The change I suggested works though.

What am I doing wrong? I'm not nearly a developer and my only exposure to working with xml has been this.